app/app.go

```diff
@@ -880,6 +890,15 @@ func (app *App) SetStoreUpgradeHandlers() {
 			Added: []string{acltypes.StoreKey},
 		}
 
+		// configure store loader that checks if version == upgradeHeight and applies store upgrades
+		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
+	}
+	// TODO: add upgrade name for authz module
+	if false && upgradeInfo.Name == "TODO" && !app.UpgradeKeeper.IsSkipHeight(upgradeInfo.Height) {
+		storeUpgrades := storetypes.StoreUpgrades{
+			Added: []string{authzkeeper.StoreKey},
+		}
```

Comment:
Why does this have to be a TODO?

Reply:
Because we don't have an upgrade version yet for which we want to add in the wrote upgrade (hence the `false` at the beginning of branch) We will need to update this when we create the next upgrade version release

Reply:
I can make a task for psu to make sure it doesnt get missed